src/util/StreamerMode.cpp

```diff
@@ -32,51 +42,108 @@ bool isInStreamerMode()
             return true;
         case StreamerModeSetting::Disabled:
             return false;
-    }
+        case StreamerModeSetting::DetectObs:
 
-#ifdef USEWINSDK
-    if (!IsWindowsVistaOrGreater())
-    {
-        return false;
-    }
-    static bool cache = false;
-    static QDateTime time = QDateTime();
+#ifdef Q_OS_LINUX
```

Comment:
Works on mac for me and i believe since MacOS 10.8 pgrep is available by default.
```suggestion
#if defined(Q_OS_LINUX) || defined(Q_OS_MACOS)
```

Reply:
I didn't want to "yolo-add" a feature while I have no way to test this on Mac. I also think `pgrep` isn't available by default on most mac installations, please correct me if I'm wrong.

Reply:
10.8 released like in 2012. So it's fair to say that most mac installation have it.

Reply:
If someone (like @pajlada or @gempir) could confirm it works on Mac, I'd be glad to make this change and get mac done as well.

Reply:
Let's leave that to a separate PR, but thanks for letting us know!